kale/embed/_mpca.py

```diff
@@ -0,0 +1,275 @@
+# =============================================================================
+# Author: Shuo Zhou, szhou20@sheffield.ac.uk
+#         Haiping Lu, h.lu@sheffield.ac.uk or hplu@ieee.org
+# =============================================================================
+
+"""Python implementation of Multilinear Principal Component Analysis (MPCA)
```

Comment:
For a reviewer, how to tell the difference from `mpca.py` (particularly this has 275 lines)? It might be better to push it as an update to mpca.py to show the difference and then rename later if that is preferred since you plan to depreciate `mpca.py` anyway.

Reply:
We can discuss if I miss anything.

Reply:
Because there were pull requests that made changes in mpca.py, and I need to resolve conficts when I merge upstream/master into the mpca branch. That's why I have both mpca.py and _mpca.py. I understand it may bring inconvinence for reivewers. I will try to merge them to one after #48 and then just keep one.

Reply:
Thanks for your thoughtfulness. That's right. #48 has been merged and I believe this is resolved now.